scipy/special/xsf/expint.h

```diff
@@ -30,25 +28,27 @@ XSF_HOST_DEVICE inline double exp1(double x) {
     constexpr double ga = cephes::detail::SCIPY_EULER;
 
     if (x == 0.0) {
-	return std::numeric_limits<double>::infinity();
+        return std::numeric_limits<double>::infinity();
     }
     if (x <= 1.0) {
         e1 = 1.0;
         r = 1.0;
         for (k = 1; k < 26; k++) {
-            r = -r*k*x/std::pow(k+1.0, 2);
+            r = -r * k * x / std::pow(k + 1.0, 2);
             e1 += r;
-            if (std::abs(r) <= std::abs(e1)*1e-15) { break; }
+            if (std::abs(r) <= std::abs(e1) * 1e-15) {
+                break;
+            }
         }
-        return -ga - std::log(x) + x*e1;
+        return -ga - std::log(x) + x * e1;
     }
-    m = 20 + (int)(80.0/x);
+    m = 20 + (int) (80.0 / x);
```

Comment:
I'd prefer not to insert a blank between `(int)` and `(80.0 / x)`, because `(int)` (reinterpret cast) acts like a function, and function names are usually not separated from the opening parenthesis. Similarly e.g. for `(std::max)(1.0, 2.0)`.

Reply:
Do you know how to change the config file to get what you are looking for?

Reply:
Is this an instance of `SpaceAfterCStyleCast: true`?

Reply:
Yes. Setting it to false should achieve what I have in mind.

Reply:
Can we hold up guys? I'm not keen on this particular change, for instance. More broadly, there is a plan to actually move `xsf` out of `special` and I think that is likely to happen soon.